.github/workflows/refapp-2x-visit.yml

```diff
@@ -0,0 +1,40 @@
+name: RefApp 2.x Visit
```

Comment:
@mherman22 do you think we can get a better name to be reflected on the dashboard?

Reply:
something like `RefApp 2.x Patient Visit`?